app/src/main/java/org/oppia/app/options/OptionsFragment.kt

```diff
@@ -0,0 +1,264 @@
+            }
+            getStoryTextSize(StoryTextSize.MEDIUM_TEXT_SIZE) -> {
+              profileManagementController.updateStoryTextSize(profileId, StoryTextSize.MEDIUM_TEXT_SIZE)
+            }
+            getStoryTextSize(StoryTextSize.LARGE_TEXT_SIZE) -> {
+              profileManagementController.updateStoryTextSize(profileId, StoryTextSize.LARGE_TEXT_SIZE)
+            }
+            getStoryTextSize(StoryTextSize.EXTRA_LARGE_TEXT_SIZE) -> {
+              profileManagementController.updateStoryTextSize(profileId, StoryTextSize.EXTRA_LARGE_TEXT_SIZE)
+            }
+          }
+        }
+
+        // Update the changed language to summary field.
+        preference.key == getString(R.string.key_app_language) -> {
+          preference.summary = stringValue
+          when (stringValue) {
+            getAppLanguage(AppLanguage.ENGLISH_APP_LANGUAGE) -> profileManagementController.updateAppLanguage(
+              profileId,
+              AppLanguage.ENGLISH_APP_LANGUAGE
+            )
+            getAppLanguage(AppLanguage.HINDI_APP_LANGUAGE) -> profileManagementController.updateAppLanguage(
+              profileId,
+              AppLanguage.HINDI_APP_LANGUAGE
+            )
+            getAppLanguage(AppLanguage.CHINESE_APP_LANGUAGE) -> profileManagementController.updateAppLanguage(
+              profileId,
+              AppLanguage.CHINESE_APP_LANGUAGE
+            )
+            getAppLanguage(AppLanguage.FRENCH_APP_LANGUAGE) -> profileManagementController.updateAppLanguage(
+              profileId,
+              AppLanguage.FRENCH_APP_LANGUAGE
+            )
+          }
+        }
+
+        // Update the changed audio language to summary field.
+        preference.key == getString(R.string.key_default_audio) -> {
+          preference.summary = stringValue
+          when (stringValue) {
+            getAudioLanguage(AudioLanguage.NO_AUDIO) -> profileManagementController.updateAudioLanguage(
+              profileId,
+              AudioLanguage.NO_AUDIO
+            )
+            getAudioLanguage(AudioLanguage.ENGLISH_AUDIO_LANGUAGE) -> profileManagementController.updateAudioLanguage(
+              profileId,
+              AudioLanguage.ENGLISH_AUDIO_LANGUAGE
+            )
+            getAudioLanguage(AudioLanguage.HINDI_AUDIO_LANGUAGE) -> profileManagementController.updateAudioLanguage(
+              profileId,
+              AudioLanguage.HINDI_AUDIO_LANGUAGE
+            )
+            getAudioLanguage(AudioLanguage.CHINESE_AUDIO_LANGUAGE) -> profileManagementController.updateAudioLanguage(
+              profileId,
+              AudioLanguage.CHINESE_AUDIO_LANGUAGE
+            )
+            getAudioLanguage(AudioLanguage.FRENCH_AUDIO_LANGUAGE) -> profileManagementController.updateAudioLanguage(
+              profileId,
+              AudioLanguage.FRENCH_AUDIO_LANGUAGE
+            )
+          }
+        }
+      }
+      true
```

Comment:
Add inline comment for this true. What is it?

Reply:
It is just Boolean.